core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -578,9 +578,19 @@ private[spark] class BlockManager(
     doGetRemote(blockId, asBlockResult = false).asInstanceOf[Option[ByteBuffer]]
   }
 
+  private def getLocations(blockId: BlockId): Seq[BlockManagerId] = {
+    /**
+     * Return a list of locations for the given block, prioritizing the local machine since
+     * multiple block managers can share the same host.
+     */
```

Comment:
Java docs should live outside the `def`, not inside. I'll fix this myself when I merge this.